processing/src/main/java/org/apache/druid/query/aggregation/SerializablePairLongStringSimpleStagedSerde.java

```diff
@@ -88,6 +102,8 @@ public SerializablePairLongString deserialize(ByteBuffer byteBuffer)
 
       readOnlyBuffer.get(stringBytes, 0, stringSize);
       lastString = StringUtils.fromUtf8(stringBytes);
+    } else if (stringSize < 0) {
```

Comment:
Shoudn't this be 
If stringSize==0 then return null
stringSize==-1 then return ""
else 
strValue?

Reply:
Oh its this only. 
Changes LGTM.